core/domain/html_cleaner.py

```diff
@@ -143,3 +144,229 @@ def get_rte_components(html_string):
             component['customization_args'] = customization_args
             components.append(component)
     return components
+
+
+def wrap_with_siblings(tag, p):
```

Comment:
Unit tests?

Reply:
This function is not returning any value and this function is called from `convert_to_text_angular`, so the tests for it also checks this function.

Reply:
Technically any function should always have its own unit tests. Unless it's private.